mason/image/plot_images.mas

```diff
@@ -0,0 +1,161 @@
+
+<%args>
+$trial_id
+$stockref => undef
+</%args>
+
+<style>
+#plot_images_results {
+    width: 100% !important;
+}
+</style>
+
+<& /util/import_javascript.mas, classes => [ 'jquery', 'thickbox', 'jquery.dataTables' ] &>
+
+
+<div class="well well-sm table-responsive">
+    <table id="plot_images_results" class="table table-hover table-striped">
+    <thead>
+        <tr>
+        <th>Image</th>
+        <th>File Name/Details</th>
+        <th>Description</th>
+        <th>Submitter</th>
+        <th>Associations</th>
+        <th>Accession Name</th>
+    </tr>
+    </thead>
+    </table>
+</div>
+            
+
+<script>
+
+<%perl>
```

Comment:
The authentication should not be needed, as the user is either unauthenticated and can't see this page, or they are already authenticataed and this shouldn't be necessary?